src/js/angular/explore/fit-text.directive.js

```diff
@@ -0,0 +1,98 @@
+angular
+    .module('graphdb.framework.explore.directives.fittext', [])
+    .directive('fitText', fitTextDirective);
+
+fitTextDirective.$inject = ['$timeout'];
+
+function fitTextDirective($timeout) {
+    return {
+        restrict: 'A',
+        scope: {
+            min: '=',
+            max: '='
+        },
+        link: function (scope, element, attrs) {
+
+            const config = {
+                'debounce': _.debounce,
+                'delay': 100,
+                'loadDelay': 10,
```

Comment:
What's the purpose of this `compressor` config. I guess this directive is borrwed from somewhere, if so, can you please add link to the source so that we can see some documentation eventually?

Reply:
Is useful when dealing with untypical custom fonts - to correct the ratio calc.
Is reworked from https://github.com/patrickmarabeas/ng-FitText.js

Reply:
Cool. Can you please add the link in the source code here?

Reply:
sure